projects/storefrontapp-e2e-cypress/cypress/integration/b2b/regression/order-history/b2b-order-history-orders-flow.e2e-spec.ts

```diff
@@ -81,4 +85,61 @@ describe('Order History with orders', () => {
     cy.get('.cx-order-history-po a').should('contain', poNumber);
     cy.get('.cx-order-history-cost-center a').should('contain', costCenter);
   });
+
+  describe('Order details - reorder', () => {
+    it('should display order details page with the reorder button', () => {
+      cy.get('.cx-order-history-value').first().click();
+      cy.get('cx-order-details-reorder div div button').should(
+        'contain',
+        'Reorder'
+      );
+    });
+
+    it('items in the cart should match previous order when proceeding with reorder', () => {
+      // Saving cartId before starting the reorder flow
+      b2bCheckout.addB2bProductToCart();
+      cart.goToCart();
+      cart.saveCartId();
+
+      goToOrderDetails();
```

Comment:
Is ther a way to know the order code from the order history test done so far and reliably go to that order's detail?

Reply:
Since the order history test creates an order, we agreed to leave it like this for now.